posix.mak

```diff
@@ -57,8 +59,6 @@ endif
 
 # build with shared library support
 # (defaults to true on supported platforms, can be overridden w/ make SHARED=0)
-SHARED=$(if $(findstring $(OS),linux freebsd),1,)
```

Comment:
Huh?

Reply:
Reverted all the posix.mak changes i made (They should not have been committed).